packages/patternfly-4/react-core/src/components/Card/CardHeadMain.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/Card/card';
+
+export interface CardHeadMainProps extends React.HTMLProps<HTMLDivElement> {
+  /** Content rendered inside the Card Head Main */
+  children?: React.ReactNode;
```

Comment:
To me it is not intuitive when this wrapper should be used.  Maybe we should add the children should be text, image or logo to make it more clear. Even if we add that a consumer could easily not include the wrapper and the styling would be off @christiemolloy @mcoker.

Reply:
Agree. I wonder if we should always have the `<CardHeadMain>` contain the contents of whatever you put in the `<CardHead>` that isn't actions? That way it's consistent. wdyt @christiemolloy?

```
<CardHead>
  <CardActions></CardActions> // optional
  <CardHeadMain>
    // images, text, <CardHeader>, etc
  </CardHeadMain>
</CardHead>
```

Reply:
Yeah it would be good if we could generate that wrapper automatically so users dont forget it. Good point!